include/editormap.hpp

```diff
@@ -11,19 +11,23 @@ class EditorMap : public Dummy::Core::GraphicMap {
     virtual ~EditorMap();
 
     Dummy::Core::GraphicLayer& firstLayer() {
-        return m_firstLayer;
+        //return m_firstLayer;
+        return m_mapLevels[0].layers().at(-1);
```

Comment:
why using .at() instead of operator[]? .at() is a lot slower compared to operator[] since it's checking if the index is out of bound or not